src/Appwrite/Utopia/Response/Model/DocumentList.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Utopia\Database\Document;
+
+class DocumentList extends BaseList
+{
+    public function filter(Document $document): Document
+    {
+        foreach ($document->getAttribute('documents', []) as $node) {
+            $node->removeAttribute('$internalId');
```

Comment:
Are we sure we need to loop here? Isn't the nested model taking care of it?

Reply:
Good catch! For some reason tests were failing locally - but shouldn't have.

Reverted that part.